apps/hash-agents/app/agents/generate-text-from-prompt/__main__.py

```diff
@@ -0,0 +1,51 @@
+import json
+
+import structlog.stdlib
+from langchain.chat_models import ChatOpenAI
+from langchain.schema import HumanMessage, SystemMessage
+
+from .io_types import Input, Output
+
+logger = structlog.stdlib.get_logger(__name__)
+
+SYSTEM_MESSAGE_CONTENT = (
+    "You are a text generator. Given a topic or a question, generate a"
+    "paragraph of text accurately describing the given topic or answering the"
+    " question. You must be concise. Do not comment on controversial issues"
+    " including politics."
```

Comment:
I don't know that we want/need the "no controversy including politics" prompt, adding another layer beyond what's already in the OpenAI model itself. I'd rather we extricate ourselves from any debate around what's permissible and what's not until we have the ability to devote dedicated resource to determining what our parameters should be - and coming up with a conscious policy, rather than making normative judgements re: what is in/out of scope beyond those already determined by the AI ethicists at our model providers (in this case OpenAI).

I'll look into how other API consumers handle this, but unless there are severe objections let's drop this for now. We'll make sure there's Additional Product ToS language around our AI-enabled features to shield against legal risks (while appreciating reputational ones may still arise, but eating them for now).